independent-publisher-connectors/Mensagia/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2330 @@
+                                    "description": "Contact's language in ISO code.",
+                                    "title": "Language",
+                                    "enum": [
+                                        "en",
+                                        "en-gb",
+                                        "en-au",
+                                        "en-in",
+                                        "en-gb-wls",
+                                        "es",
+                                        "ca",
+                                        "zh-cn",
+                                        "zh-tw",
+                                        "da",
+                                        "nl",
+                                        "fr",
+                                        "de",
+                                        "it",
+                                        "ja",
+                                        "ko",
+                                        "no",
+                                        "pl",
+                                        "pt-pt",
+                                        "pt-br",
+                                        "ru",
+                                        "sv",
+                                        "fi",
+                                        "tr",
+                                        "wls"
+                                    ],
+                                    "default": "en"
+                                },
+                                "groups": {
+                                    "type": "string",
+                                    "description": "Add your contact to an address book",
+                                    "title": "Agenda",
+                                    "x-ms-dynamic-values": {
+                                        "operationId": "get_agendas",
+                                        "value-path": "id",
+                                        "value-collection": "data",
+                                        "value-title": "name",
+                                        "parameters": {
+                                            "Accept": "application/json",
+                                            "Api-source": "Power-Automate"
+                                        }
+                                    }
+                                }
+                            },
+                            "required": [
+                                "email",
+                                "number"
+                            ]
+                        }
+                    }
+                ]
+            }
+        }
+    },
+    "definitions": {},
+    "parameters": {},
+    "responses": {},
+    "securityDefinitions": {
+        "Clave de API": {
+            "type": "apiKey",
+            "in": "query",
+            "name": "api_token"
+        }
+    },
+    "security": [{
+        "Clave de API": []
```

Comment:
@AfriSR We do not support this security type, you can remove the entire security section.